src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+  const costSmallOff = cost - 20;
+  const costBigOff = cost - 50;
+
+  if (days >= 3 && days < 7) {
+    return costSmallOff;
+  } else if (days >= 7) {
+    return costBigOff;
+  } else {
+    return cost;
+  }
```

Comment:
```suggestion
  } 
  
  if (days >= 7) {
    return costBigOff;
  } 
  
  return cost;
```
try to avoid else if where it possible. it make code more readable